shell/src/main/java/alluxio/shell/command/TestCommand.java

```diff
@@ -100,25 +169,22 @@ public void test(AlluxioURI path, boolean testDir, boolean testFile, boolean tes
   }
 
   @Override
-  public void run(CommandLine cl) throws AlluxioException, IOException {
-    String[] args = cl.getArgs();
-    AlluxioURI path = new AlluxioURI(args[0]);
-    test(path, cl.hasOption("d"), cl.hasOption("f"), cl.hasOption("e"),
-         cl.hasOption("s"), cl.hasOption("z"));
-  }
-
-  @Override
   public String getUsage() {
     return "test [-d|-f|-e|-s|-z] <path>";
   }
 
   @Override
   public String getDescription() {
-    return "Display properties of the path."
-        + " Specify -d to test whether the path is a directory."
-        + " Specify -f to test whether the path is a file."
-        + " Specify -e to test whether the path exists."
-        + " Specify -s to test whether the directory is not empty."
-        + " Specify -z to test whether the file is zero length.";
+    return "Test properties of the path."
+        + " Specify -d to test whether the path is a directory,"
+        + " returning 0 if it's true or 1 if it's false."
```

Comment:
I think we can just say the 0/1 thing once at the start, i.e. "Tests a property of a path, returning 0 if the property is true, and 1 otherwise"